### DISCUSSION TITLE
How to configure python version?

### DISCUSSION THREAD

**Discussion by vitarsi** [tobi-wan-kenobi/bumblebee-status#893] [Q&A] [answered]
I have both python 2.7.17 and python 3.6.9 installed. However, bumblebee-status gives the error "at least Python 3.4 required (Python 2.x not supported)"

How can I configure which python is used?

**Comment by tobi-wan-kenobi** [comment, marked as answer]
bumblebee-status uses the system-wide default (i.e. if you enter python3 --version) in the CLI, that is the version used.

Which version are you on, maybe it is one that still uses python instead of python3 (i.e. the system-wide default for python, which can be either 2 or 3)
